app/models/rbac.rb

```diff
@@ -445,7 +445,7 @@ def self.search(options = {})
   def self.method_with_scope(ar_scope, options)
     if ar_scope == VmdbDatabaseConnection
       ar_scope.all
-    elsif ar_scope < ActsAsArModel
+    elsif ar_scope < ActsAsArModel || ar_scope.respond_to?(:instances_are_derived?) && ar_scope.instances_are_derived?
```

Comment:
I would add parentheses to make the complex condition clearer:

`elsif ar_scope < ActsAsArModel || (ar_scope.respond_to?(:instances_are_derived?) && ar_scope.instances_are_derived?)`